src/App.tsx

```diff
@@ -1,12 +1,39 @@
 import "@mantine/core/styles.css";
 import { MantineProvider } from "@mantine/core";
 import { theme } from "@/theme";
+import Navbar from "@/components/Navbar";
+import SearchBar from "@/components/SearchBar";
+import "@/css/App.css";
+import DataTable from "@/components/DataTable";
 import PreviewCard from "./components/PreviewCard";
```

Comment:
Can we make this fit in with the rest 
My neat freak disorder be bugging!

Reply:
Clarity and Readability: Using @ can make it clear that the path is rooted in a specific directory (often the src folder), which can make the code easier to read.

Simplified Imports: It can reduce the need for long relative paths (e.g., ../../../components/Navbar), making imports more straightforward and less error-prone.

Consistency: Having a uniform import style across the project can help maintain consistency, making it easier for new developers to understand the structure.

Easier Refactoring: If the project structure changes, you may only need to update the base alias rather than all the relative paths, simplifying refactoring.